shell/packages/accounts-identity/accounts-identity-server.js

```diff
@@ -238,3 +202,62 @@ Accounts.linkIdentityToAccount = function (identityId, accountId) {
                       {$push: {"nonloginIdentities": {id: identityId}}});
 
 }
+
+Meteor.publish("accountsOfIdentity", function (identityId) {
+  check(identityId, String);
+  var hasIdentityCursor =
+      Meteor.users.find({$or: [{_id: identityId},
```

Comment:
The first clause in this `or` seems wrong -- it doesn't in any way depend on the caller's user ID, and I think it will always match at least one user record if the identity is in the system at all.